Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]] # Package source
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages] # Packages required to run the application
+requests = "==2.23.0"
+
+[dev-packages] # Packages required to develop the application
+coverage = "==5.0.3"
+pylint = "==2.4.4"
+responses = "==0.10.12"
+flit = "==2.2.0"
+azure-identity = "==1.3.1"
```

Comment:
Latest version of azure-identity is @1.0.5

Reply:
Thanks. I'll update to the latest version